src/App.vue

```diff
@@ -100,6 +101,14 @@ const init = () => {
     component: markRaw(NodeLibrarySidebarTab),
     type: 'vue'
   })
+  app.extensionManager.registerSidebarTab({
+    id: 'model-library',
+    icon: 'pi pi-folder-open',
```

Comment:
We can probably pick another icon. Maybe `pi-box`?

Reply:
yeah wasn't sure what icon would work here. Swapped to box